999-storage-import-zipped-vhds/azuredeploy.json

```diff
@@ -0,0 +1,173 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.2105082",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location in which the resources should be deployed."
+            }
+        },
+        "Source": {
+            "type": "string",
+            "metadata": {
+                "description": "URL of a ZIP archive containing VHD images files. For example: https://provider.com/bin/support/azure-vhd.zip"
+            }
+        },
+        "Destination": {
+            "type": "string",
+            "metadata": {
+                "description": "URI of a Storage Account container with a writable SAS Token. For example: https://blob-storage-name.blob.core .windows.net/container-name?sp=racwl&st=2021-01-01T12:00:00&se=2021-02-01T12:00:00&spr=https&sv=2021-01-01&sig=123412341234"
+            }
+        }
+    },
+    "functions": [],
+    "variables": {
+        "urlVHDZipArchive": "[parameters('Source')]",
+        "uriWritableStorageAccountBlobContainerSasToken": "[parameters('Destination')]",
+        "nestedTemplateName_ImportVHD-FromZipArchive": "[concat('ImportVHD-FromZipArchive-',uniqueString(parameters('Source'),uniqueString(parameters('Destination'))))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
```

Comment:
why is this nested?

Reply:
Also i am using the nested template as a modular function and I did couple of others template on the same model. I would prepare some other PR when that one is aligned with the quickstart best practices.

In this quickstart template, it is embedded surfacing only few parameters to provide an easy use experience in Azure Portal when launching the quickstart. An other way to use it is just to embed like in this example https://github.com/riverbed/Riverbed-Community-Toolkit/tree/master/NetIM/Azure-Cloud-Cookbooks/101-netim-simple-demo.

Reply:
The nesting doesn't change the UX, you have the same parameters defined in both deployments.  I'm not seeing any reason why you need the nested deployment so just remove it.

Reply:
Done.
It is now straight without nesting and the UX will show Source and Destination user parameters, and also location and forceupdatetag "internal" parameters i wanted to hide but that is fine. I have added some details in the README.